### PR TITLE
add some tiflash tune instructions

### PR DESCRIPTION
**Pull request by hanfei1991**
### What is changed, added or deleted? (Required)

TiFlash 调优建议 section

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@hanfei1991 请 involve 技术 review。

**Review by ilovesoup — APPROVED** [review]
lgtm

File: tiflash/tune-tiflash-performance.md [hunk 1]
```diff
@@ -50,3 +50,13 @@ aliases: ['/docs-cn/dev/tiflash/tune-tiflash-performance/','/docs-cn/dev/referen
     ```sql
     set @@tidb_opt_distinct_agg_push_down = 1;
     ```
+
+5. 在 Join 没有选择 MPP 执行模式的情况下，调整参数使 Join 算子选择 MPP 执行：
```
Comment (TomShawn) [inline tiflash/tune-tiflash-performance.md:54]:
```suggestion
5. 如果 `Join` 算子没有选择 MPP 执行模式，你可以调整 `tidb_opt_network_factor` 变量值使 `Join` 算子选择 MPP 执行模式：
```

File: tiflash/tune-tiflash-performance.md [hunk 2]
```diff
@@ -50,3 +50,13 @@ aliases: ['/docs-cn/dev/tiflash/tune-tiflash-performance/','/docs-cn/dev/referen
...
+
+    `tidb_opt_network_factor` 变量用来设置优化器计算代价时考虑网络开销代价的比例。当这个值越小时，TiDB 对于大量网络传输的代价估算就更小，从而更倾向于选择 MPP 算子。
```
Comment (TomShawn) [inline tiflash/tune-tiflash-performance.md:56]:
```suggestion
    `tidb_opt_network_factor` 变量用来设置优化器计算代价时考虑网络开销的比例。该变量值越小，TiDB 对于大量网络传输的开销估算就越小，从而更倾向于选择 MPP 算子。
```

**Comment by TomShawn** [comment]
lgtm

**Pull request merged**